src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -18,10 +18,10 @@ public abstract class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Adds an entity (person or appointment) to the address book. "
+            + ": Adds an entity to address or appointment book. \n"
             + "Parameters: "
-            + "ENTITY_TYPE "
-            + "ENTITY_ARGUMENTS..."
+            + "ENTITY_TYPE (person/appt) "
+            + "DATA_FIELDS [MORE_DATA_FIELDS]... \n"
             + "Example: " + COMMAND_WORD + " person "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
```

Comment:
Should we include another example for adding an appointment as well? i.e. include examples for both person and appt for `EditCommand`, `FindCommand`, `DeleteCommand`, `ClearCommand`, `ListCommand`?